src/components/Settings/Settings.tsx

```diff
@@ -15,60 +26,85 @@ function Settings() {
-          <div className="pl-[6%] pr-[8%] font-sans text-2xl">
-            <Input
-              className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4"
-              size="lg"
-            />
-          </div>
-          <div className="pl-[4%] pt-[3%] pl-2 text-xl"> Option </div>
-          <div className="pl-[6%] pr-[8%] font-sans text-2xl">
-            <Input
-              className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4"
-              size="lg"
-            />
-          </div>
-          <div className="pl-[4%] pt-[3%] pl-2 text-xl"> Option </div>
-          <div className="pl-[6%] pr-[8%] font-sans text-2xl">
-            <Input
-              className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4"
-              size="lg"
-            />
-          </div>
-          <div className="pl-[4%] pt-[3%] pl-2 text-xl"> Option </div>
-          <div className="pl-[6%] pr-[8%] font-sans text-2xl">
-            <Input
-              className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4"
-              size="lg"
-            />
-          </div>
+          <form onSubmit={handleSubmit}>
+            <label className="pl-[4%] pl-2 text-xl"> Device ID </label>
+            <div className="pl-[6%] pr-[8%] font-sans text-sm">
+              <Input
+                className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4 disabled:opacity-80"
+                type="text"
+                size="lg"
+                label="Unique preset identifier for this device."
+                value={deviceID}
+                onChange={(e) => setDeviceID(e.target.value)}
+                disabled
+              />
+            </div>
+            <label className="flex pl-[4%] pt-[6%] pl-2 text-xl">
+              {" "}
+              Device Name{" "}
+            </label>
+            <div className="pl-[6%] pr-[8%] font-sans text-sm">
+              <Input
+                className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4"
+                type="text"
+                size="lg"
+                label="Personalised name for this device."
+                value={deviceName}
+                onChange={(e) => setDeviceName(e.target.value)}
+              />
+            </div>
+            <label className="flex pl-[4%] pt-[6%] pl-2 text-xl">
+              {" "}
+              Device Nickname{" "}
+            </label>
+            <div className="pl-[6%] pr-[8%] font-sans text-sm">
+              <Input
+                className="rounded-lg bg-gray-300 hover:bg-gray-200 p-4"
+                type="text"
+                size="lg"
+                label="3 character nickname for small screens."
+                maxLength={3}
+                minLength={1}
+                value={deviceNickname}
+                onChange={(e) => setDeviceNickname(e.target.value)}
+              />
+            </div>
+            <label className="flex pl-[4%] pt-[6%] pl-2 text-xl">
+              {" "}
+              Option 4{" "}
```

Comment:
What is this a placeholder for? Is it meant for a specific feature (e.g. renaming nodes)? Do we have a specific reason to include it?

Reply:
Gotcha, seems fine to leave it then. I commented on it because it looked like code that could turn into a stub without any meaning. To avoid that, mind doing some combination of (1) expressing the extra options by name in the display itself, (2) documenting them in the code comments and/or (3) adding tickets to the board to finalize them?